src/test/resources/emails/sessionUnpublishedEmailForInstructor.html

```diff
@@ -0,0 +1,16 @@
+Hello Instructor1 Course1,
+<p/>
+The email below has been sent to students of course: idOfTypicalCourse1.<p/><br><br>=== Email message as seen by the students ===<br>
```

Comment:
this is generated using GodMode. To do this, I have to update string in EmailGenerator.java. Should I do that ? Otherwise we can leave this as it is ?